typed-racket-lib/typed-racket/base-env/prims-contract.rkt

```diff
@@ -448,7 +448,7 @@
                       ;The actual identifier bound to the constructor
                       [real-maker (if (syntax-e #'id-is-ctor?) #'internal-maker #'maker-name)]
                       [extra-maker (and (attribute input-maker.extra)
-                                        (not (bound-identifier=? #'make-name #'nm))
+                                        (not (bound-identifier=? #'maker-name #'nm))
```

Comment:
Is this the only substantive change, other than removing the `type-only` field everywhere?